botocore/signers.py

```diff
@@ -196,6 +208,65 @@ def generate_presigned_url(self, request_dict, expires_in=3600,
         return request.url
 
 
+class CloudFrontSigner(object):
```

Comment:
Ah that's true. Will you be able to reuse much code such that it makes sense for them to be the same class?